README.md

```diff
@@ -1,12 +1,43 @@
 # Trieste
 
-A Bayesian optimization toolbox built on [TensorFlow](https://www.tensorflow.org/). Trieste supports Python 3.7 onwards and uses [semantic versioning](https://semver.org/).
+[![PyPI](https://img.shields.io/pypi/v/trieste.svg)](https://pypi.org/project/trieste)
+[![License](https://img.shields.io/badge/license-Apache-green.svg)](LICENSE)
+[![Quality checks](https://github.com/secondmind-labs/trieste/actions/workflows/quality-checks.yaml/badge.svg)](https://github.com/secondmind-labs/trieste/actions?query=workflows%3Aquality-checks)
```

Comment:
Is this the badge we want here? AFAICT it will show red whenever any unmerged PR is failing some tests. Maybe we want https://github.com/secondmind-labs/trieste/actions/workflows/develop-checks.yaml/badge.svg instead?

Reply:
ok, found a way to show the names nicely, we need to add a name field to our workflows - I'll do that

I agree that quality-checks doesn't make sense as it run on every PR, but develop-checks though runs only the slow tests so that doesn't seem appropriate as well 

actually, what we need is checks on master branch, running everything we have, thats what matters here, right?

Reply:
develop is essentially master in trieste, so develop-checks is the closest we have (though I could extend it to also rerun the quick tests, even though these need to have already passed for you to merge)

Reply:
we often had issues with develop checks not passing, it seems bad that anytime that something is merged we would get failing signs here, while the official release is healthy - I think we simply need another GH action that runs everything when there is a release, i.e. something really merged to master